src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Http/Convert.ts

```diff
@@ -23,7 +23,7 @@ var sanityCheck = (obj : ResourcesBase.IResource) : void => {
         throw ("resource has no content_type field:\n" + JSON.stringify(obj, null, 2));
     }
 
-    if (!Resources_.resourceRegistry.hasOwnProperty(obj.content_type)) {
+    if (!adhMetaApi.resourceExists(obj.content_type)) {
```

Comment:
Yes. `Resource_.ts` will still get generated, but not used anywhere. This also means that we will no longer download all generated files. (Example: on master, the demo frontpage will load 301KB in 125 generated files; in this branch, the same page will load 184KB in 73 generated files.)

I plan to remove `mkResources.ts` in its current incarnation anyway, so I did not spend time to stop it from generating `Resources_.ts`.